docs/core/tools/dotnet-sln.md

```diff
@@ -28,6 +28,12 @@ dotnet sln [-h|--help]
 
 The `dotnet sln` command provides a convenient way to add, remove, and list projects in a solution file.
 
```

Comment:
nit: "you need to have a solution file already" -> the solution file must already exist

Reply:
Makes it sooo much better. Thanks @rpetrusha!